terraso_backend/apps/storage/file_utils.py

```diff
@@ -0,0 +1,22 @@
+# Copyright © 2021-2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+def is_file_upload_oversized(files, max_size):
+    for file in files:
+        return get_file_size(file) > max_size
```

Comment:
suggestion: `get_file_size` is unnecessary. Reference `file.size` directly.

Reply:
@paulschreiber We need the get_file_size function to mock an oversized file when unit testing: Check `terraso_backend/tests/shared_data/test_views.py`